tasks/README.md

```diff
@@ -268,8 +293,8 @@ Task output is consumed on stdout and made available through the task runner API
 If the output cannot be parsed as a JSON object (i.e. `{...}`), one is created by the task runner with the captured stdout stored in the  `_output` key: i.e. `{"_output": "the string of output"}`. Otherwise the parsed JSON object is used.
 
 All '_' prefixed keys are reserved and should only be used as described below:
-- **_output**: A text summary of the job result. If the job does not return a JSON object the contents of stdout will be put in this key.
-- **_error**: Tasks can set this when they fail and the UI can more gracefully display messages from it.
+- **_output**: A text summary of the job result. If the job does not return a JSON object the contents of stdout will be put in this key. (rev 1)
```

Comment:
In these lines you put a space before `(rev N)`, unlike everywhere else. I'd like that to be consistent throughout the doc.